ex05.js

```diff
@@ -0,0 +1,22 @@
+// Read the file and stream it to client as text
+let fs = require('fs');
+
+const express = require('express');
+const app = express();
+app.use(express.static('./public'));
+
+app.get('/files/', (req, res) => {
+
+    let fileName = req.query.filename || null;
+   
+    // Make sure file exists
+    if (!(fs.existsSync(`./public/${fileName}`))) {
```

Comment:
This is good but it is open for something called "path traversal", read about it a little bit.